base/applications/utilman/umandlg/lang/en-US.rc

```diff
@@ -51,4 +41,6 @@ END
 STRINGTABLE
 BEGIN
     IDM_ABOUT "About Accessibility Utility Manager..."
+    IDS_APP_NAME "Accessibility Utility Manager"
+    IDS_AUTHORS "Copyright 2019 George Bișoc, 2019 Hermes Belusca-Maito"
```

Comment:
Could shorten this to `Copyright 2019 George Bișoc, Hermes Belusca-Maito` but either way seems fine to me.